### ISSUE TITLE
cli: renku status

### ISSUE THREAD

**Issue by LiliGasser** [SwissDataScienceCenter/renku-python#432]
## Describe the bug
A clear and concise description.

## Details
*Please verify and redact the details.*

**Renku version:** 0.3.4.dev30
**OS:** Linux (48~16.04.1-Ubuntu SMP Tue Jan 29 18:03:48 UTC 2019)
**Python:** 3.6.5

### Traceback

```
Traceback (most recent call last):
  File "[...]/site-packages/renku/cli/_graph.py", line 349, in build_status
    can_be_cwl=can_be_cwl,
  File "[...]/site-packages/renku/cli/_graph.py", line 320, in build
    self.process_dependencies(dependencies, visited=ignore)
  File "[...]/site-packages/renku/cli/_graph.py", line 284, in process_dependencies
    client=processing.client,
  File "[...]/site-packages/renku/models/provenance/activities.py", line 225, in from_git_commit
    return from_git_commit(commit, client, path=None)
  File "[...]/site-packages/renku/models/provenance/activities.py", line 584, in from_git_commit
    raise ValueError(file_)  # duplicate
ValueError: .renku/workflow/7b69fad35c574cbfb28b966576580e0d_python.cwl
```

## Additional context
Add any other context about the problem.

**Comment by LiliGasser** [comment]
`renku status` on a jupyter server with the same project at the same state gives the same error message:
```
jovyan@jupyter-gasserli-luis-2d2esal-2ddemocrasci-2da9e4327:~/democrasci_preprocwp1$ renku --version
0.3.2.post2.dev12
jovyan@jupyter-gasserli-luis-2d2esal-2ddemocrasci-2da9e4327:~/democrasci_preprocwp1$ renku status
Traceback (most recent call last):
  File "/home/jovyan/.renku/bin/renku", line 11, in <module>
    sys.exit(cli())
  File "/opt/conda/envs/renku/lib/python3.6/site-packages/click/core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "/opt/conda/envs/renku/lib/python3.6/site-packages/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/opt/conda/envs/renku/lib/python3.6/site-packages/click/core.py", line 1137, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/opt/conda/envs/renku/lib/python3.6/site-packages/click/core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/opt/conda/envs/renku/lib/python3.6/site-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "/opt/conda/envs/renku/lib/python3.6/site-packages/renku/cli/_client.py", line 75, in new_func
    result = ctx.invoke(method, client, *args, **kwargs)
  File "/opt/conda/envs/renku/lib/python3.6/site-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "/opt/conda/envs/renku/lib/python3.6/site-packages/click/decorators.py", line 17, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/opt/conda/envs/renku/lib/python3.6/site-packages/renku/cli/status.py", line 63, in status
    status = graph.build_status(revision=revision, can_be_cwl=no_output)
  File "/opt/conda/envs/renku/lib/python3.6/site-packages/renku/cli/_graph.py", line 349, in build_status
    can_be_cwl=can_be_cwl,
  File "/opt/conda/envs/renku/lib/python3.6/site-packages/renku/cli/_graph.py", line 320, in build
    self.process_dependencies(dependencies, visited=ignore)
  File "/opt/conda/envs/renku/lib/python3.6/site-packages/renku/cli/_graph.py", line 284, in process_dependencies
    client=processing.client,
  File "/opt/conda/envs/renku/lib/python3.6/site-packages/renku/models/provenance/activities.py", line 225, in from_git_commit
    return from_git_commit(commit, client, path=None)
  File "/opt/conda/envs/renku/lib/python3.6/site-packages/renku/models/provenance/activities.py", line 584, in from_git_commit
    raise ValueError(file_)  # duplicate
ValueError: .renku/workflow/7b69fad35c574cbfb28b966576580e0d_python.cwl
jovyan@jupyter-gasserli-luis-2d2esal-2ddemocrasci-2da9e4327:~/democrasci_preprocwp1$
```

**Comment by jirikuncar** [comment]
fixed in ecf5180
